plutus-core/flat/flat.cabal

```diff
@@ -0,0 +1,171 @@
+cabal-version:      3.0
```

Comment:
The flat code is independent of the rest of the Plutus code at the moment.  I wonder if we should put it in a separate package, like `plutus-flat` or something.  I don't know if that would be sensible or not, but it seems like the sort of thing that people mighr have an opinion about.

Reply:
Why are the other libs like satint in `plutus-core` instead of separate packages?